Jellyfin.Plugin.Tvdb/TvdbClientManager.cs

```diff
@@ -477,7 +477,7 @@
                     break;
             }
 
-            key = $"FindTvdbEpisodeId_{seriesTvdbIdString}_{seasonNumber.Value.ToString(CultureInfo.InvariantCulture)}_{episodeNumber.Value.ToString(CultureInfo.InvariantCulture)}";
+            key = $"FindTvdbEpisodeId_{seriesTvdbIdString}_{seasonNumber.Value.ToString(CultureInfo.InvariantCulture)}_{episodeNumber.Value.ToString(CultureInfo.InvariantCulture)}_{searchInfo.SeriesDisplayOrder}";
```

Comment:
Won't be null, all cases in the switch case will aways assign non null value to episodeNumber as IndexNumber.HasValue must be true before it can hit this line of code